solr/core/src/java/org/apache/solr/handler/admin/api/InvokeClassAPI.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.handler.admin.api;
+
+import org.apache.solr.api.Command;
+import org.apache.solr.api.EndPoint;
+import org.apache.solr.api.PayloadObj;
+import org.apache.solr.client.solrj.request.beans.InvokeClassPayload;
+import org.apache.solr.handler.admin.CoreAdminHandler;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import static org.apache.solr.client.solrj.SolrRequest.METHOD.POST;
+import static org.apache.solr.common.params.CoreAdminParams.ACTION;
+import static org.apache.solr.common.params.CoreAdminParams.CoreAdminAction.INVOKE;
+import static org.apache.solr.handler.ClusterAPI.wrapParams;
+import static org.apache.solr.security.PermissionNameProvider.Name.CORE_EDIT_PERM;
+
+/**
+ * V2 API for triggering "invocable" classes.
+ *
+ * This API (POST /v2/node {'invoke': {...}}) is analogous to the v1 /admin/cores?action=INVOKE command.
```

Comment:
Umm, https://solr.apache.org/guide/8_9/coreadmin-api.html doesn't mention INVOKE.....    Is that something we need to add to the ref guide?   Or is it some low level "end users don't touch it code"?    I wonder if end users aren't meant to touch it, maybe we put those under /v2/node/internal/???       And lastly, is there a chance we can get rid of whatever INVOKE does?

Reply:
I'm not sure.  Prior to this PR I'd never heard of 'invoke' either.  It seems like an internal detail, but since there's no existing documentation or even comment in the Java code and this is the first I've stumbled on it, I'm a little leery to make that call here.

I can do some JIRA or git-blame digging and try to find out.

Either way though, I'm not sure I've got the context to even really know what to say about it in the ref-guide.  I suspect I hate that as much as you do, but the only thing worse than no-docs is wrong-docs 😬